src/prompt.rs

```diff
@@ -2,10 +2,17 @@ use clipboard::{ClipboardContext, ClipboardProvider};
 use rpassword;
 use std::fs;
 use std::io::{stdin, stdout, Result, Write};
-use std::path::Path;
 use std::thread;
 use std::time::Duration;
-use crate::admin::VAULT_PATH;
+
+use std::env;
+use std::path::PathBuf;
+
+pub fn vault_path() -> PathBuf {
+    let username = env::var("USER").unwrap_or_else(|_| "default_user".to_string());
+    let vault_path = format!("/home/{}/.vault", username);
+    PathBuf::from(vault_path)
+}
```

Comment:
great implementation!
It would be better if we stored the vault in /var/lib/.vault since it has limited permissions.
I will do that, you have done enough, love this.

Reply:
Oh true. Thank you!
I'd love to take on this issue also https://github.com/db-keli/kelvin/issues/2